tests/columns/test_low_cardinality.py

```diff
@@ -151,3 +151,10 @@ def test_fixed_string(self):
 
             inserted = self.client.execute(query)
             self.assertEqual(inserted, data)
+
+    def create_table(self, columns, **kwargs):
```

Comment:
It's better to use [client_kwargs](https://github.com/mymarilyn/clickhouse-driver/blob/master/tests/columns/test_decimal.py#L11) method.